app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+from pathlib import Path
+from shutil import move
+
+
+def move_file(command: str) -> None:
+    input_command, source_path, move_path = command.split()
```

Comment:
you code will raise error without any info if command will be one of:
mv file1.txt
mv file1.txt file2.txt file3.txt